CMakeLists.txt

```diff
@@ -747,7 +752,7 @@ else()
 endif()
 list(APPEND LIBS ${ICONV_LIBRARIES})
 
-if(NOT ANDROID)
+if(NOT ANDROID AND HAVE_UDEV)
```

Comment:
Please don't.
If udev is required, add it to the block below where it is used and make it possible for the block not to force enable HIDAPI then.

I believe that udev isn't really available on ALL the targets we have anyway, so the test is wrong.